code/modules/mob/living/carbon/human/species.dm

```diff
@@ -470,7 +482,7 @@ GLOBAL_LIST_EMPTY(features_by_species)
 		for(var/obj/item/organ/external/organ_path as anything in external_organs)
 			//Load a persons preferences from DNA
 			var/obj/item/organ/external/new_organ = SSwardrobe.provide_type(organ_path)
-			new_organ.Insert(human)
+			new_organ.Insert(human, TRUE, FALSE)
```

Comment:
use named args here

Reply:
I've now done this both here and in several other places in this file.